src/main/resources/application.yml

```diff
@@ -8,9 +8,9 @@ spring:
       accept-single-value-as-array: true
   elasticsearch:
     rest:
-      uris: http://elasticsearch:9200
+      uris: http://${ELASTICSEARCH_HOST:elasticsearch}:${ELASTICSEARCH_PORT:9200}
```

Comment:
maybe better to use elasticsearch_url parameter? in case we would have external elasticsearch url with https protocol

Reply:
Make sense, but I've agreed the ENV variables names with devops and they already added them to the helm charts. Let's postpone the change until we have a case, then we can change it to a single property?